portability-transfer/src/main/java/org/dataportabilityproject/transfer/PortabilityTransferServiceProviderRegistry.java

```diff
@@ -84,4 +86,26 @@ public PortabilityTransferServiceProviderRegistry(List<String> enabledServices,
     Preconditions.checkArgument(serviceProvider != null);
     return serviceProvider.getImporter(transferDataType);
   }
+
+  /**
+   * Returns the set of service ids that can transfered for the given {@code transferDataType}.
+   *
+   * @param transferDataType the transfer data type
+   */
+  @Override
+  public Set<String> getServices(String transferDataType) {
```

Comment:
I am getting deja vu all over again... saw this code in another place too, is that intentional?

(not trying to be obnoxious, just like quoting Yogi Berra..)

Reply:
Ha! I was also getting major dejavu and realized it's because this PR includes changes from #134 which is making it a bit difficult to read.